openstackinabox/models/keystone/model.py

```diff
@@ -314,11 +345,15 @@ def init_database(self):
         # Create an admin user and add the admin token to that user
         self.__admin_tenant_id = self.add_tenant('system',
                                                  'system administrator')
+        self.add_admin_tenant_details()
```

Comment:
One of two things should happen:
(i) L#348 is called before L#346, and then the dictionary created by L#348 is used for the parameters
(ii) L#348 and its associated function simply go away

Given that the tenantid is stored, ii is probably the right solution.